### DISCUSSION TITLE
How to reduce the memory occupation of clickhouse-serv？

### DISCUSSION THREAD

**Discussion by limuzi2231** [ClickHouse/ClickHouse#38612] [General]
I am using the Clickhouse with version 20.8.3.18. The data in the current data table is about 300MB, but after the Clickhouse serv has been running for a period of time, the memory  occupation of process clickhouse-serv has reached 3-4GB. How can I reduce the memory occupation of the clickhouse-serv? Thanks a lot.

**Comment by nikitamikhaylov** [comment]
Usually mark cache is the root cause of such an issues and it may look like memory leak. Try to execute `SYSTEM DROP MARK CACHE`.

**Reply by limuzi2231** [reply]
Thanks, let me have a try.
